pkg/fuse/fuse.go

```diff
@@ -0,0 +1,106 @@
+package fuse
+
+import (
+	"os"
+	"context"
+	"github.com/golang/glog"
+	"github.com/kubevirt/device-plugin-manager/pkg/dpm"
+	pluginapi "k8s.io/kubernetes/pkg/kubelet/apis/deviceplugin/v1beta1"
+	"fmt"
+	"time"
+)
+
+const (
+	FUSEPath 		 = "/dev/fuse"
+	FUSEName  		= "fuse"
+	Namespace 		= "devices.kubevirt.io"
+	fusePoolSize 	= 128
+)
+
+type message struct{}
+
+type FusePlugin struct {
+	counter int
```

Comment:
the counter is not needed anymore, is it?

Reply:
No I will remove it